pkg/smclient/client.go

```diff
@@ -379,6 +380,11 @@ func (client *serviceManagerClient) DeletePlatforms(q *query.Parameters) error {
 	return err
 }
 
+func (client *serviceManagerClient) CascadeDeletePlatform(id string, q *query.Parameters) (string, error) {
```

Comment:
why `CascadeDeletePlatform`? isn't this simply `DeletePlatform`? the cascade behaviour is enabled by the param from the caller, not by this function

Reply:
Done, renamed